script/hassfest/requirements.py

```diff
@@ -126,8 +123,6 @@ def validate_requirements(integration: Integration) -> None:
                 f"Failed to normalize package name from requirement {req}",
             )
             return
-        if (package == ign for ign in IGNORE_PACKAGES):
```

Comment:
The reason for this check is that packages need to be installed for the check to work. The commented packages may not be able to be installed. If that has changed, I don't know.